packages/core/src/config/config.ts

```diff
@@ -104,6 +108,7 @@ export class StyleguideConfig {
       [SpecVersion.Async2]: { ...rawConfig.decorators, ...rawConfig.async2Decorators },
       [SpecVersion.Async3]: { ...rawConfig.decorators, ...rawConfig.async3Decorators },
       [SpecVersion.Arazzo1]: { ...rawConfig.arazzo1Decorators },
```

Comment:
should this line include `...rawConfig.decorators`?

Reply:
Most likely yes. @DmitryAnansky what do you think?

Reply:
these can be added in an separate pr unrelated to Overlays.. I found this omission when updating this pr.

Reply:
Sure, thanks.